hydro_deploy/core/src/ssh.rs

```diff
@@ -299,18 +344,41 @@ impl<T: LaunchedSshHost> LaunchedHost for T {
                         10,
                         Duration::from_secs(1),
                     )
-                    .await?;
+                    .await
+                }
+                // Launch with perf if specified, also copy local binary to expected place for perf report to work
+                let perf_wrapper = if let Some(perf) = perf.clone() {
+                    {
+                        let mut channel = open_channel(&session).await?;
+                        channel.exec("sudo sh -c 'apt update && apt install -y linux-perf && echo -1 > /proc/sys/kernel/perf_event_paranoid && echo 0 > /proc/sys/kernel/kptr_restrict'").await?;
+                        channel.send_eof().await?;
+                        channel.wait_eof().await?;
+                        channel.wait_close().await?;
+                    }
```

Comment:
I've changed it to run this command instead of needing a custom image - works with stock debian now - probably want a better way to do VM setup though

Reply:
Switched to use `metadata_startup_script`